server/models/Request.js

```diff
@@ -0,0 +1,42 @@
+import mongoose from ("mongoose");
+import dbConnection from "./../db/mongoose";
+
+const RequestSchema = mongoose.Schema(
+    {
+        userId: [
+            {
+                type: mongoose.Schema.Types.ObjectId,
+                ref: 'User',
+                required: true
+            }
+        ],
+        requestedUserId: [
+            {
+                type: mongoose.Schema.Types.ObjectId,
+                ref: 'User',
+                required: true
+            }
+        ],
```

Comment:
What's the point of square brackets? Are those arrays?

Reply:
To be honest, I'm not clear about this. I saw Schema.Types.ObjectId is arrays in some tutorial, so I put them in arrays. But if it doesn't matter, I'll take it off

Reply:
Yeah, I would remove them